### PR TITLE
Create monitor-off.svg

### PR DESCRIPTION
**Pull request by yukosgiti**
Added Monitor Off icon.

### CODE REVIEW

**Comment by locness3** [comment]
Welcome to Lucide, thanks for your contribution! Please see https://github.com/lucide-icons/lucide/blob/master/docs/ICON_DESIGN_GUIDE.md - especially about formatting the svg file.

File: icons/monitor-off.svg [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  stroke="currentColor"
+  stroke-width="2" 
+  stroke-linecap="round" 
+  stroke-linejoin="round">
+    <path d="M2 15C2 16.1046 2.89543 17 4 17H17L3 3C3 3.5 2 3.5 2 5V15Z"/>
+    <path d="M22 15V5C22 3.89543 21.1046 3 20 3H17H9"/>
+    <path d="M8 21H16"/>
+    <path d="M12 17V21"/>
+    <path d="M1 1L23 23"/>
+</svg>
```
Comment (ericfennis) [inline icons/monitor-off.svg:16]: @yukosgiti Nice work, love the icon.
I have some minor changes for your icon. To follow the guidelines.
The crossing stroke need to be 1px of the canvas. 

```suggestion
<svg
  xmlns="http://www.w3.org/2000/svg"
  width="24"
  height="24"
  viewBox="0 0 24 24"
  fill="none"
  stroke="currentColor"
  stroke-width="2"
  stroke-linecap="round"
  stroke-linejoin="round"
>
  <path d="M17 17H4a2 2 0 01-2-2V5c0-1.5 1-2 1-2" />
  <path d="M22 15V5a2 2 0 00-2-2H9" />
  <path d="M8 21h8" />
  <path d="M12 17v4" />
  <path d="M2 2l20 20" />
</svg>
```

**Comment by yukosgiti** [comment]
@ericfennis Thanks!

**Comment by ericfennis** [comment]
@yukosgiti Nice work!

**Pull request merged**